internal/nginx/convert_test.go

```diff
@@ -101,7 +101,7 @@ func TestGetMapKeyAsInt(t *testing.T) {
 		"key": "123456789",
 	}
 
-	i, exists, err := GetMapKeyAsInt(configMap.Data, "key", &configMap)
+	i, exists, err := GetMapKeyAsInt64(configMap.Data, "key", &configMap)
```

Comment:
You're testing `TestGetMapKeyAsInt` while the test func name is still  `TestGetMapKeyAsInt`and refers to `GetMapKeyAsInt`.